src/main/java/newbank/server/NewBank.java

```diff
@@ -13,17 +16,23 @@ private NewBank() {
   }
 
   private void addTestData() {
-    Customer bhagy = new Customer("Bhagy", "bhagy");
-    bhagy.addAccount(new Account("Main", 1000.0));
-    customers.put("Bhagy", bhagy);
+    try {
+      Customer bhagy = new Customer("Bhagy", "bhagy");
+      bhagy.addAccount(new Account("Main", 1000.0));
+      customers.put("Bhagy", bhagy);
+
+      Customer christina = new Customer("Christina", "christina");
+      christina.addAccount(new Account("Savings", 1500.0));
+      customers.put("Christina", christina);
+
+      Customer john = new Customer("John", "john");
+      john.addAccount(new Account("Checking", 250.0));
+      customers.put("John", john);
+    } catch (AccountInvalidNameException e) {
```

Comment:
Perhaps just one "catch(Exception e)" would do as the behaviour is the same in both cases

Kindest Regards,
Jules